library/ns1_zone.py

```diff
@@ -135,28 +168,26 @@
 
 
 try:
-    from ns1 import NS1, Config
     from ns1.rest.errors import ResourceException
 except ImportError:
     # This is handled in NS1 module_utils
     pass
 
-ZONE_KEYS = [
-    'refresh',
-    'retry',
-    'expiry',
-    'nx_ttl',
-    'ttl',
-    'link',
+
+# list of keys that should be treated like a set() during diff
+SET_KEYS = [
+    'other_ips',
+    'other_ports',
     'networks',
-    'secondary',
-    'secondary_enabled',
-    'primary_ip',
-    'primary_port',
-    'primary',
-    'primary_enabled',
-    'secondaries',
-    'dnssec'
+]
+
+# list of keys that should be ignored for calls to API
```

Comment:
can we make this comment more precise about what "ignored" means? So future spelunkers don't have to think about it. maybe `list of keys that will be removed from the data passed to API` or something